frontend/public/components/storage-class-form.tsx

```diff
@@ -882,7 +900,8 @@ export type StorageClass = {
   metadata: object,
   provisioner: string,
   parameters: object,
-  reclaimPolicy?: string
+  reclaimPolicy?: string,
```

Comment:
I missed this on last review, but the type declarations should use `;` instead of `,`